### DISCUSSION TITLE
How do I add/enable support for dark-mode with `@tailwindcss/forms`?

### DISCUSSION THREAD

**Discussion by Svish** [tailwindlabs/tailwindcss#11458] [Help]
I have a pretty basic Tailwind setup, with `darkMode: 'media'`, and have installed the `@tailwindcss/forms` plugin.

Added an input field, with no classes, and this is what I get by default:
![light mode](https://github.com/tailwindlabs/tailwindcss/assets/142162/d6e88de9-e4d1-43fa-85c8-a3cad9de9b6a)
![dark mode](https://github.com/tailwindlabs/tailwindcss/assets/142162/8124d521-eb1e-4381-8f0e-20a886e2e150)

Is there a config or switch to enable dark mode for the plugin? And whether there is such a thing or not, what is the best/correct way to add or _override_ plugin related colors for the input elements in light mode and dark mode?

Would be great if info about this was in the plugin readme.

**Comment by wongjn** [comment]
There is no dark mode "switch" or configuration. You should be able to use utility classes on the elements for dark mode, like any other element:

```html
<input type="text" class="dark:bg-slate-950"/>
```

**Reply by Svish** [reply]
Oh, OK, so basically all colors are just default ones? The plugin actually just does non-color related stuff?

**Reply by wongjn** [reply]
The plugin does set colors, as can be [seen in the source code](https://github.com/tailwindlabs/tailwindcss-forms/blob/master/src/index.js).

**Reply by Svish** [reply]
Hm, as a Tailwind-newb it would be great if it was documented how to best provide your own colors for the plugin (in light and dark mode) then. How to do it, which colors exists that should be supplied, and so on.
